db/migrations/20190128181133_initial.js

```diff
@@ -1,8 +1,34 @@
 
 exports.up = function(knex, Promise) {
-  
+  return Promise.all([
+    knex.schema.createTable('cities', (table) => {
+      table.increments('id').primary()
+      table.string('name')
+      table.string('state')
+      table.integer('population')
+      table.boolean('capital')
+
+      table.timeStamps(true, true)
+    }),
+
+    knex.schema.createTable('restaurants', (table) => {
+      table.increments('id').primary()
+      table.string('name')
+      table.string('address')
+      table.string('city')
+      table.integer('rating')
+      table.string('avg_cost')
+      table.foreign('cities_id')
```

Comment:
I think we need a table.integer('city_id'), and table.foreign() should have city_id inside instead of cities_id.

Reply:
Good catch, thanks! Just fixed